go.mod

```diff
@@ -1,43 +1,96 @@
-module github.com/chainflag/eth-faucet
+module github.com/LiskHQ/lsk-faucet
 
-go 1.17
+go 1.21
```

Comment:
Please consider updating the Golang version to `1.22`. This is in contradiction with the README specifications stating the min required Golang version is `1.22 or above`.